podvm/Dockerfile.podvm

```diff
@@ -13,27 +13,29 @@ FROM ${BINARIES_IMG} AS podvm_binaries
 FROM ${BUILDER_IMG} AS podvm_builder
 
 ARG CLOUD_PROVIDER
+ARG SE_BOOT
 ARG PODVM_DISTRO=ubuntu
 ARG UEFI=false
 
 # If not provided, uses system architecture
 ARG ARCH=x86_64
 ARG CAA_SRC=""
-ARG CAA_SRC_REF=""
+ARG CAA_SRC_BRANCH=""
```

Comment:
You can use either branch name or commit id for `CAA_SRC_REF`

Reply:
Got it, thanks @bpradipt  so that I will keep using `CAA_SRC_REF ` and then update related Document.